Gems/AtomLyIntegration/AtomFont/Code/Source/FFont.cpp

```diff
@@ -1534,7 +1535,16 @@ void AZ::FFont::Prepare(const char* str, bool updateTexture, const AtomFont::Gly
         UpdateTexture();
         m_fontTexDirty = false;
         ++m_fontImageVersion;
-
+        /*
```

Comment:
This produces a set of images to visually track how the font grows, it helps to set proper texture size.